portlets/opencps-portlet/docroot/WEB-INF/src/org/opencps/dossiermgt/portlet/ConfigurationImpl.java

```diff
@@ -52,7 +52,11 @@ public void processAction(
 	    throws Exception {
 
 		long plid = ParamUtil.getLong(actionRequest, "plid");
-
+		
+		String displayDossierNo = ParamUtil.getString(actionRequest, "displayDossierNo");
+		
+		String displayRecentlyResultWhenSearch = ParamUtil.getString(actionRequest, "displayRecentlyResultWhenSearch");
+		
```

Comment:
Sao lại có cái này sửa vào đây?

Đây là cấu hình cho thanh toán keypay, trước đã nói vấn đề này trên telegram rồi cơ mà?